app/controllers/people_controller.rb

```diff
@@ -16,21 +16,27 @@ def orgchart
     @page_title = "Org Chart"
     render :layout => "orgchart"
   end
+  
+  def department
+    dept = Department.find(params[:dept_id])
+    # TODO: more gracefully handle department not found
+    head 404 and return if dept.nil?
+    
+    @people = Person.active.where(department_id: dept.id)
+    @page_title = dept.name
+    render :template => "people/index"
+  end
 
   def police
-    @people = Person.active.select{|person| person.department&.name == "Police"}
-    @page_title = "Police"
-    render :template => "people/index"
+    redirect_to department_people_path("Police")
```

Comment:
this doesnt work, as the parameter to this route is an id, not a shortname as I originally intended. I thought the shortnames would work fine, but since shortnames arent immutable it felt a bit weird that changing a shortname would break links to a specific department's people page.